src/include/add-on/add-on-workflow.rb

```diff
@@ -1505,11 +1497,9 @@ def RemoveProductWithDependencies
         return nil
       end
 
-      product_name = Ops.get_locale(
-        pi,
-        ["product", "display_name"],
-        Ops.get_locale(pi, ["product", "name"], _("Unknown product"))
-      )
+      product_name = [ pi["product"].display_name,
```

Comment:
This logic is basically the same than line 1454. What about having a method to do it?